### DISCUSSION TITLE
【测试】发布一个帖子试一下

### DISCUSSION THREAD

**Discussion by ncwsszs** [share121/inter-knot#3386] [其他]
![88072716_p0](https://github.com/user-attachments/assets/a40b47b8-6b09-4170-af55-446c94ec51e5)

# 作者动手能力好强啊

测试下发帖，马上就删除<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by Lainisang** [comment]
怎么删除

**Reply by ncwsszs** [reply]
删除不了，我已经关闭了还是会显示，估计是作者没有筛选吧
